.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: node_js
+
+node_js:
+  - "0.10"
+  - "4"
+
+sudo: false
+
+cache:
+  directories:
+    - node_modules
+
+notifications:
+  irc:
+    channels:
+      - 'irc.mozilla.org#fxa-bots'
+    use_notice: false
+    skip_join: false
```

Comment:
This seems important enough what we should add email notifications on failure, they'd be easily missed in IRC.